### PR TITLE
README: Drop scala 2.11

### PR DESCRIPTION
**Pull request by htmldoug**
Reflects 083ace92a951f093afb77ec129181574a9771a0a.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ support package.
 
 ### Quick Start
 
-Jawn supports Scala 2.11, 2.12, and 2.13.0-RC1.
+Jawn supports Scala 2.12, 2.13.0, and dotty.
```
Comment (travisbrown) [inline README.md]: Can you change `2.13.0` to `2.13`? The patch version isn't relevant now that 2.13 is out.

Also, it's probably worth clarifying that Jawn isn't currently published for Dotty, just cross-built with it.

Comment (travisbrown) [reply README.md]: Also we should capitalize Dotty if we mention it.

**Comment by htmldoug** [comment]
Wow, thanks for the fast response! Updates complete. If you want to hold on README updates, I wouldn't be offended if you want to just close it, too. Totally up to you.

**Comment by travisbrown** [comment]
Ha, Jawn is on my mind this week, with 1.0.0 just a few days away. I'm happy to merge this now but let's give the other maintainers a chance to take a look.

**Pull request merged**